package-lock.json

```diff
@@ -15389,6 +15354,125 @@
         "@pkgjs/parseargs": "^0.11.0"
       }
     },
+    "node_modules/jake": {
```

Comment:
noting that this is needed for ts-jest for some reason. hopefully we can remove that soon after cleaning up the coordinator.